### PR TITLE
Fix changelog.sh for Windows

### PR DESCRIPTION
**Pull request by 3y3**
Previous version doesn't work correctly (it miss first commit message in changelog list).
I can't understand this behavior, but this commit fix it.

### CODE REVIEW

**Comment by 3y3** [comment]
@bajtos , I'm ready to merge this one and release 0.8.2

Also I'm glad to notice that now we have prebuild binaries for Linux and OSX x64 (I'm ready to add x32 atoreleasing if it's needed)

File: tools/git-changelog.sh [hunk 1]
```diff
@@ -8,9 +8,7 @@ tmp="$TMPDIR/changelog"
 
 if [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ]; then
   NULL="NUL";
-  # TMP was overriden by Bash (TMP="/tmp") we need to unset it.
-  unset TMP;
-  tmp="$TMP/changelog"
+  tmp="._changelog"
```
Comment (bajtos) [inline tools/git-changelog.sh]: What is the purpose of this change?

I am not entirely against keeping the temp file in the local directory. However, it must be listed in `.gitignore` in such case, to ensure it is not accidentally committed to git and/or released to npmjs.org. The name `.changelog` would be better in such change.

Comment (3y3) [inline tools/git-changelog.sh]: First `.gitignore` rule is `._*`, I get the name relative to this rule. Correct me if understand incorrect this mask.

**Comment by bajtos** [comment]
> First .gitignore rule is ._*, I get the name relative to this rule. Correct me if understand incorrect this mask.

Oh, I did not realise we have such rule in `.gitignore`.

Let's not dwell on details and ship this patch then.

LGTM.
